src/pages/en/guides/migrate-to-astro/from-nextjs.mdx

```diff
@@ -2,72 +2,924 @@
 title: Migrating from Next.js
 description: Tips for migrating an existing Next.js project to Astro
 layout: ~/layouts/MigrationLayout.astro
-stub: true
+stub: false
 framework: Next.js
 ---
-
+import AstroJSXTabs from '~/components/tabs/AstroJSXTabs.astro'
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
-[Next.js](https://nextjs.org) is a framework for building web applications on top of React.
+Here are some key concepts and migration strategies to help you get started. Use the rest of our docs and our [Discord community](https://astro.build/chat) to keep going!
 
-## Key Similarities
+## Key Similarities between Next.js and Astro
 
 Next.js and Astro share some similarities that will help you migrate your project:
 
-- [Astro's JSX-like syntax](/en/core-concepts/astro-components/#the-component-template) should feel familiar if you are used to writing React.
-
-- Both Next.js and Astro are modern Javascript static-site generators. Both use a [`src/` folder for your project files](/en/core-concepts/project-structure/#src) and a [special folder for file-based routing](/en/core-concepts/astro-pages/). Creating and managing pages for your site should feel familiar.
-
-- Astro has an [official integration for using React components](/en/guides/integrations-guide/react/). Note that in Astro, React files **must** have a `.jsx` or `.tsx` extension.
-
-- Astro supports [installing NPM packages](/en/guides/imports/#npm-packages), including several for React. You may be able to keep some or all of your existing React components and dependencies.
-
-
-- Astro projects can also be SSG or [SSR](/en/guides/server-side-rendering/). (Support for per-page rendering strategy is planned.)
-
+- The [syntax of `.astro` files is similar to JSX](/en/core-concepts/astro-components/#differences-between-astro-and-jsx). Writing Astro should feel familiar.
+- Astro projects can also be SSG or [SSR with page level prerendering](/en/guides/server-side-rendering/#prerendering).
```

Comment:
In case it is in fact "page-level", we also need to change here.
```suggestion
- Astro projects can also be SSG or [SSR with page-level prerendering](/en/guides/server-side-rendering/#prerendering).
```